src/reducers/index.js

```diff
@@ -30,7 +31,10 @@ import { NewSensorReducer } from './NewSensorReducer';
 import SyncReducer from './SyncReducer';
 import { SensorDetailReducer } from './SensorDetailReducer';
 
+const BluetoothReducer = combineReducers({ blink: BlinkReducer });
```

Comment:
Nit: Maybe should be done in a `BluetoothReducer` or the `Bluetooth/index` file? Maybe only because I navigate and find things through file names 🤷

Reply:
Oh yup good idea, nicer to have the Bluetooth stuff aggregated closer to the actual files too I think? 4615bb4a